include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -792,18 +773,6 @@ class FEEvaluationBase
    */
   const MatrixFree<dim, Number, VectorizedArrayType> *matrix_info;
```

Comment:
Even though this line has not been changed, the name `matrix_info` is also unclear in my opinion.